qemu/tests/virtio_fs_share_data.py

```diff
@@ -533,6 +535,24 @@ def file_check(cmd):
                     if session.cmd_status(cmd_symblic_folder):
                         test.fail("Creat symbolic folders failed.")
                     if os_type == "linux":
+                        error_context.context("Compare symbolic link info in "
+                                              "the host and guest", test.log.info)
+
+                        def __file_check(file, guest=None):
+                            if guest:
+                                o = session.cmd_output("ls -l %s" % file)
+                            else:
+                                o = process.run("ls -l %s" % file).stdout_text
+                            return o.strip().split()[-1]
+
+                        if __file_check(file_link, 'guest') != \
+                                __file_check(os.path.join(fs_source, file_link)):
```

Comment:
```suggestion
                        if (__file_check(file_link, 'guest') !=
                                __file_check(os.path.join(fs_source, file_link))):
```
I prefer this way of wrapping, refer to https://peps.python.org/pep-0008/#maximum-line-length:
`The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.`

Reply:
Many thanks for your review, I learned a lot.
Corrected.